src/utils/index.js

```diff
@@ -9,21 +9,14 @@
  */
 
 import docTypes from './frontend/form-helper';
-import runNotification from './frontend/notification-helper';
+import { runNotification, clearProcessing } from './frontend/notification-helper';
 import {
   uploadDocument,
   updateDocument,
   getDocuments,
   deleteDocumentContainer,
   deleteDocumentFile,
   setDocAclPermission,
-  generateUsersList,
```

Comment:
Should keep this for the usersList.

Reply:
This functionality has been moved to `loadUserList` in `model-helpers/UserList.js`